thunder-service/service/oci_metrics.go

```diff
@@ -241,20 +315,23 @@ func (as *ThunderService) CountEventsOccurence(client monitoring.MonitoringClien
 	var instance Instance
 	var cnt int
 	for _, s := range resp.Items {
+		// reset the counter
+		cnt = 0
```

Comment:
If you need to reset the counter, why don't you initialize it here instead of initialize it outside at line 316?

Reply:
Because the reset is needed for each iteration.

Reply:
Made optimization